imagersite/imager_profile/templates/profile.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+{% block content %}
+
+<div class='imager-profile'>
+<p id='join_date'>Joined on {{ user.date_joined|date:"D d M Y" }}</p>
+<h2>{{ user.first_name }} {{ user.last_name }}</h2>
```

Comment:
look at `user.get_full_name`, or even the `__str__` method of your profile class.